dojo/api_v2/serializers.py

```diff
@@ -1121,8 +1121,12 @@ def save(self, push_to_jira=False):
                 item.reporter = self.context['request'].user
                 item.last_reviewed = timezone.now()
                 item.last_reviewed_by = self.context['request'].user
-                item.active = data['active']
-                item.verified = data['verified']
+
+                if item.active:
+                    item.active = data['active']
+                if item.verified:
+                    item.verified = data['verified']
+
```

Comment:
I think this will work, maybe we should put a small comment in here to explain "Only set active / verified flags if they were not changed by the parser from their default value (True)"?

Also we need to add the same logic to the UI import.